src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -447,6 +447,18 @@ public string TargetedRuntimeVersion
             set { _targetedRuntimeVersionRawValue = value; }
         }
 
+        /// <summary>
+        /// If not null, serializes a cache to this location. This overrides the usual cache, so only use this if you will
+        /// not have access to the usual cache at the next build.
```

Comment:
```suggestion
        /// If not null, computes and information about <see cref="AssemblyFiles" /> inputs to the named file.
        /// This overrides the usual outputs, so do not use this unless you are building an SDK with many references.
```

Reply:
I took out "computes and" because it would do that whether or not this is set but took the rest. Thanks for the suggestion!